install_ffmpeg.sh

```diff
@@ -130,7 +135,7 @@ if [ ! -e "$HOME/ffmpeg/libavcodec/libavcodec.a" ]; then
     --enable-bsf=h264_mp4toannexb,aac_adtstoasc,h264_metadata,h264_redundant_pps,extract_extradata \
     --enable-parser=aac,aac_latm,h264 \
     --enable-filter=abuffer,buffer,abuffersink,buffersink,afifo,fifo,aformat \
-    --enable-filter=aresample,asetnsamples,fps,scale \
+    --enable-filter=aresample,asetnsamples,fps,scale,lvpdnn \
```

Comment:
Should we conditionally enable `lvpdnn` based on the `experimental` tag as well?

Reply:
The `vf_lvpdnn.c` file currently has the definition of the model loading functions, whose declaration is in `avfilter.h` that we always include.
Not enabling this filter lead to compiler errors for undefined method.

Reply:
But agreed, once we land up on a better solution for model loading on the ffmpeg side - we can conditionally include that filter.

Reply:
Gotcha thanks for the clarification.